test/python/tools/monitor/test_backend_monitor.py

```diff
@@ -20,11 +20,15 @@
 
 from qiskit.tools.monitor import backend_overview, backend_monitor
 from qiskit.test import QiskitTestCase, requires_qe_access
+from qiskit.util import _has_connection
+# Check if internet connection exists
+HAS_NET_CONNECTION = _has_connection('qiskit.org', 443)
 
 
 class TestBackendOverview(QiskitTestCase):
     """Tools test case."""
 
+    @unittest.skipIf(not HAS_NET_CONNECTION, "requries internet connection.")
```

Comment:
```suggestion
    @unittest.skipIf(not HAS_NET_CONNECTION, "requires internet connection.")
```

Reply:
I fixed this typo in https://github.com/Qiskit/qiskit-terra/pull/2448